src/Console/Application.php

```diff
@@ -286,10 +290,19 @@ public function buildContainer()
         };
 
         // VIOLATION RENDERER
-        $c['violation.renderer'] = function ($c) {
+        $c['violation.renderer.console'] = function ($c) {
             return new Renderer\ConsoleOutputRenderer(new ConsoleOutput(), $c['violation.message_helper']);
         };
 
+        $c['violation.renderer.html'] = function ($c) {
+            return new Renderer\HtmlOutputRenderer(
+                new ConsoleOutput(),
+                $c['violation.message_helper'],
+                new Filesystem(),
+                './output.html'
```

Comment:
The intention was to not overwrite the interface method with optional parameters. If somebody will change the interface and add a parameter, tho component will not behave as expected. Do you have a suggestion to bypass that problem?